kikit/panelize_ui_impl.py

```diff
@@ -253,18 +256,31 @@ def buildLayout(preset, panel, sourceBoard, sourceArea):
                 vbonefirst=layout["vbonefirst"])
             substrates = panel.makeGrid(
                 boardfile=sourceBoard, sourceArea=sourceArea,
-                rows=layout["rows"], cols=layout["cols"], destination=VECTOR2I(0, 0),
+                rows=layout["rows"], cols=layout["cols"], destination=layout["destination"],
                 rotation=layout["rotation"], placer=placer,
                 netRenamePattern=layout["renamenet"], refRenamePattern=layout["renameref"],
                 bakeText=layout["baketext"])
             framingSubstrates = dummyFramingSubstrate(substrates, preset)
             panel.buildPartitionLineFromBB(framingSubstrates)
             backboneCuts = buildBackBone(layout, panel, substrates, framing)
             return substrates, framingSubstrates, backboneCuts
+        if type == "manual":
+            board_list = [sourceBoard] + layout["boards"]
```

Comment:
I don't like baking the source boards into a preset. It seems to be unsystematic, and it also leaves the users with confusing UI:
- you specify all locations in the preset,
- you specify the first board as the argument,
- and the rest in preset.